doc/api/addons.md

```diff
@@ -1364,6 +1364,7 @@ console.log(result);
 [Embedder's Guide]: https://github.com/v8/v8/wiki/Embedder's%20Guide
 [Linking to libraries included with Node.js]: #addons_linking_to_libraries_included_with_node_js
 [Native Abstractions for Node.js]: https://github.com/nodejs/nan
+[V8]: https://github.com/v8/v8
```

Comment:
This URL is a mirror of https://chromium.googlesource.com/v8/v8.git -- in particular contributions to V8 do not go via GitHub so it would probably be better to link to the googesource.com URL or to the project page at https://v8.dev/.

cc @nodejs/v8-update